engines/director/lingo/xlibs/jwxini.cpp

```diff
@@ -58,21 +68,19 @@ const char *JourneyWareXINIXObj::fileNames[] = {
 };
 
 static MethodProto xlibMethods[] = {
-	{"dispose", 					JourneyWareXINIXObj::m_dispose, 0, 0, 400}, // D4
 	{"new", 						JourneyWareXINIXObj::m_new, 0, 0, 400}, // D4
-	{"name", 						JourneyWareXINIXObj::m_name, 0, 0, 400}, // D4
-	{"getPrivateProfileInt", 		JourneyWareXINIXObj::m_GetPrivateProfileInt, 4, 4, 400}, // D4
```

Comment:
Why are you making these case changes? Please revert.

Reply:
Its a revert of the changes introduced here: https://github.com/scummvm/scummvm/commit/8ba2bd487995f642eaa408f4a9a09762da7699f0

You asked me to revert them.

Reply:
Apologies, I remember that I asked you to revert them, but somehow I thought they were named "getPrivateProfileInt", not "GetPrivateProfileInt". So yes, your previous revert was correct :/

Reply:
> So yes, your previous revert was correct :/

So is it fine now? :)